### PR TITLE
Removed .d-checkbox__description* and .d-radio__description* classes

### PR DESCRIPTION
**Pull request by juliodialpad**
## Description
We created global form description text classes (.d-description), but Checkbox and Radio still use their own (.d-checkbox_description and .d-radio_description). All three classes represent the same styles.

We didn't include this change in 6.0.0 as it was a breaking change from the existing 5.x Checkbox and Radio classes. So we want to re-visit this and clean up the duplication when we have more time.

https://switchcomm.atlassian.net/browse/DT-20?atlOrigin=eyJpIjoiZmE4NGUyZDI1ZmNkNDVjZTg2NTQ5MmNhOTk3NTRjYTIiLCJwIjoiaiJ9

## Pull Request Checklist

 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Provide a description what is being changed, extended, or removed.
 - [x] Link to any issue(s) this request is resolving.

### CODE REVIEW

**Review by ryandialpad — APPROVED** [review]

**Comment by ryandialpad** [comment]
@juliodialpad You will also need to update the documentation

**Review by ryandialpad — CHANGES_REQUESTED** [review]
Please update radio and checkbox docs

**Comment by juliodialpad** [comment]
> Please update radio and checkbox docs

Sure, I'll do it now, thanks

File: docs/_data/checkbox.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
       "description": "Styles checkbox group in a disabled state."
     },
     {
-      "class": "d-checkbox__description",
+      "class": "d-description",
       "applies": "N/A",
       "description": "Styles checkbox's description text."
     },
```
Comment (ryandialpad) [inline docs/_data/checkbox.json]: We can just remove this entry now

File: docs/_data/radio.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
       "description": "Styles radio group in a disabled state."
     },
     {
-      "class": "d-radio__description",
+      "class": "d-description",
       "applies": "N/A",
       "description": "Styles radio's description text."
     },
```
Comment (ryandialpad) [inline docs/_data/radio.json]: We can just remove this entry now

**Review by ryandialpad — APPROVED** [review]
w/ comments

**Pull request merged**